pawn/Features/Gameplay/VIP/VeryImportantPlayersManager.pwn

```diff
@@ -54,7 +63,7 @@ class VeryImportantPlayersManager {
             SetPlayerWeather(playerId, -66);
             TimeController->setPlayerOverrideTime(playerId, 12, 0);
         } else {
-            SendClientMessage(playerId, Color::Information, "Usage: /my weather [day/evening/hot/storm/foggy/drugs]");
+            SendClientMessage(playerId, Color::Information, "Usage: /my weather [day/evening/underwater/sandstorm/drugs/hot/cloudy/storm/foggy]");
```

Comment:
Same for this, try to keep the order same and add the new ones last or go for an alphabetical order

Reply:
Done :)